src/transformations/fix-import-directives.ts

```diff
@@ -5,41 +5,60 @@ import { Transformation } from './type';
 import { renameContract, renamePath } from '../rename';
 import { TransformerTools } from '../transform';
 
-export function* fixImportDirectives(
-  ast: SourceUnit,
-  { resolver }: TransformerTools,
-): Generator<Transformation> {
-  for (const imp of findAll('ImportDirective', ast)) {
-    const referencedSourceUnit = resolver.resolveNode('SourceUnit', imp.sourceUnit);
-
-    const aliases = imp.symbolAliases.map(a => {
-      const id = referencedSourceUnit.exportedSymbols[a.foreign.name]?.[0];
-      if (id === undefined) {
-        throw new Error(`Can't find symbol imported in ${ast.absolutePath}`);
+declare module '../transform' {
+  interface TransformData {
+    importPath: string;
+  }
+}
+
+export function fixImportDirectives(withPeerProject?: boolean) {
+  return function* (
+    ast: SourceUnit,
+    { resolver, getData }: TransformerTools,
+  ): Generator<Transformation> {
+    for (const imp of findAll('ImportDirective', ast)) {
+      const referencedSourceUnit = resolver.resolveNode('SourceUnit', imp.sourceUnit);
+
+      if (withPeerProject && imp.symbolAliases.length == 0) {
+        throw new Error(
+          `Transpile with peer doesn't support import without aliases in ${imp.absolutePath}`,
+        );
+      }
+
+      const imports: Record<string, string[]> = {};
+
+      for (const a of imp.symbolAliases) {
+        const id = referencedSourceUnit.exportedSymbols[a.foreign.name]?.[0];
+        if (id === undefined) {
+          throw new Error(`Can't find symbol imported in ${ast.absolutePath}`);
+        }
+
+        const contract = resolver.resolveContract(id);
+        const forceImport = contract && getData(contract).importPath;
```

Comment:
```suggestion
        const forceImport = contract && getData(contract).importPath !== undefined;
```
Do we want that to be explicit ?

Reply:
Yes, we should always explicitly compare with undefined as a best practice.